cowbird/request_task.py

```diff
@@ -0,0 +1,40 @@
+from abc import ABC
+
+from celery import Task
+from requests.exceptions import RequestException
+
+
+class AbortException(Exception):
+    """
+    Exception raised when the chain must be interrupted.
+    """
+
+
+class RequestTask(Task, ABC):
+    """
+    Celery base task that should be used to handle API requests.
+
+    Using this class will set the following Task configuration :
+     - autoretry for every RequestException
+     - backoff and jitter strategy
+    There is also an abort_chain function to stop the chain of requests in case of an unrecoverable event
+
+    To use this class simply decorate your asynchronous function like this :
+
+    @shared_task(bind=True, base=RequestTask)
+    def function_name(self, any, wanted, parameters):
```

Comment:
Indent this and use `::` above to have auto-formatting and code highlighting applied.